NewForce-Cohort-4/tabloid-full-stack-coral
# Post Details

# Description

The post details component allows the user to view the details for an individual post. The details include post title, , header image, content, publication date, and the author's display name.

## Type of change


# How To Test This Component

Run the application and go to /post/1 to view the details for the post with an id of 1. Try out different IDs to ensure it is working properly. 

# Checklist:
